src/index.ts

```diff
@@ -28,6 +28,7 @@ export { Text, MarkedText } from './components/Text';
 export { TextArea } from './components/TextArea';
 export { PageWidthRestrictor, BlockWidthRestrictor } from './components/WidthRestrictor';
 export { PopupBase } from './components/PopupBase';
+export { Tag } from './components/Tag';
```

Comment:
can you move it up so that it is alphabetically ordered? Or maybe that is in the other index file, in storybook, to make sure that it is in the right position in the left side of storybook.

Reply:
Implemented.

Reply:
Its in different import, but still - implemented.